### ISSUE TITLE
Disc 5 problem 1.g)

### ISSUE THREAD

**Issue by sxysun-zz** [dsherma7/Math.UCDavis#1]
I think the answer would be like exp(-x/2)
Why would there be no minus sign?

**Comment by dsherma7** [comment]
You're right, it should be e^(-x/2), I was thinking e^x had an alternator. Good catch!
